runtime/compiler/optimizer/VectorAPIExpansion.cpp

```diff
@@ -1449,9 +1449,24 @@ TR_VectorAPIExpansion::isVectorizedOrScalarizedNode(TR::Node *node, TR::DataType
    int32_t refId = -1;
 
    if (node->getOpCodeValue() == TR::aload ||
-       node->getOpCodeValue() == TR::astore)
-      {
-      refId = node->getSymbolReference()->getReferenceNumber();
+       node->getOpCodeValue() == TR::astore ||
+       (node->getOpCode().isVectorOpCode() &&
+        (node->getOpCode().getVectorOperation() == TR::vload ||
+         node->getOpCode().getVectorOperation() == TR::vstore)
```

Comment:
With `vload` and `vstore` handled here, does that mean that `origSymRef->getSymbol()->isMethod()` is now guaranteed in the `isVectorOpCode()` case below?

Reply:
Yes, I think we could put an assert that it's a method now but somehow I would prefer not to do it at this moment.